### PR TITLE
Pokedex - Module 2 - Group Capstone Project

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### JavaScript capstone project - Your API-based webapp ( Module 2 )
 - Followed the project requirements and completed the 3 days Milestones of **"Your API-based webapp"** we completed the main requirements:
   - A home page showing a list of items that you can "like." ✅
   - A popup window with more data about an item that you can use to comment on it. ✅
   - Professional Documentation in a Readme file. ✅
- As part of the requirements, we recorded a video explaining how the page works. 
   - You can watch it here: [Loom Video - Pokedex - Module 2 - Group Capstone](https://www.loom.com/share/1446ee58f4574b658372ef24167a1272) ✅

**Preview : 🕵🏼‍♂️**

final_preview_pokedex

### CODE REVIEW

File: src/modules/pokemons.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          .then((src) => {
+            const card = `<div class="card">
+            <img class="pokemon-img" id=${id} src="${src}" alt="Pokemons">
+            <div class="card-title">
+            <h2>${pokemonName}</h2>
+            <img id="likeId${id + 1}" class="likeBtn" src=${unlike} alt="like_icon">
+            </div> 
+            <p class="like-count" id="like-count${id + 1}">0 Likes</p>
+            <div class="buttons">
+              <button type="button" id="${id + 1}" class="comment">Comments</button>
```
Comment (Lucas-Masaba) [inline src/modules/pokemons.js:40]: - [ ] I am able to see the comments only on the mobile view. I cannot see them on the desktop view. Even on the mobile view, I find it hard to read the comments. Please ensure that the comments written can be seen clearly as seen in the wireframe :+1: 

### View of comments on the mobile screen
image

### View of comments on the desktop view
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @JuanDiegoGuerra  and @lincoln1883  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks and unit tests are passing :heavy_check_mark: 
- Good use of ES6 :surfer: 
- Well-written commit messages :+1: 
- Successfully used an API :snowboarder: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made by @JuanDiegoGuerra & @lincoln1883  🛠
**All linters are enable!**

- Added Media Query ✅
- Fixed _"I am able to see the comments only on the mobile view. I cannot see them on the desktop view. Even on the mobile view, I find it hard to read the comments. Please ensure that the comments written can be seen clearly as seen in the wireframe 👍"_ ✅
  - Now the app is working correctly!

todolist1

**Preview 🕵🏼‍♂️:**

requested_changes_preview

**Review by jerryowusu — APPROVED** [review]
Hi @JuanDiegoGuerra and @lincoln1883 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


weldone


## Highlights
Nice design :+1: 
API working fine :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**